src/masonite/notification/NotificationManager.py

```diff
@@ -0,0 +1,146 @@
+"""Notification handler class"""
+import uuid
+from masoniteorm.models import Model
+
+from ..utils.collections import Collection
+from ..drivers.notification.BaseDriver import BaseDriver
+from ..exceptions.exceptions import NotificationException, DriverNotFound
+from ..queues import ShouldQueue
+
+
+class NotificationManager(object):
```

Comment:
We don't have to call it `NotificationManager`. If we base off the other features I just did a simple class like `Notification`.

Reply:
Actually I called it this way because I have a Notification class which represent a notification that can be sent

Reply:
I can rename everything to send `Message` instead of `Notification` and then I can call the feature class `Notification` but I don't know if it's better. Everyone know what a Notification, I am not sure if we should call it another way. 
🤔